app/actions/action_user.js

```diff
@@ -20,7 +20,7 @@ export const updateFriends = (payload) => {
 //we have array of all friends but fb does not give photo on initial call, so we have another
 //helper function that gets each users friends photo
 const generateFriends = (friends, token, callback) =>{
-  let friendsArray = [];
+  let friendsObj = {};
   _.each(friends, function(friend, index) {
     let photoquery =  "https://graph.facebook.com/v2.3/"+friend.id+"?fields=picture,name&access_token="+token;
```

Comment:
Querying FB may be more idiomatic, but we do already have user profile pic urls in Firebase from when they logged in.